pilco/__init__.py

```diff
@@ -1,3 +1,5 @@
 from . import models
-from . import controllers
-from . import rewards
\ No newline at end of file
+# from . import controllers
+# from . import rewards
+# from . import utils
+# from . import safe_pilco
```

Comment:
Why did you exclude this imports? Were they slowing us down?

Reply:
Yup, these should go back in, I think I excluded them while upgrading, to test modules that were ready before these were done.